vevestaX/vevesta.py

```diff
@@ -302,9 +303,35 @@ def __profilingReport(self, fileName):
              }
         ]
         profilingDataframe = pandas.DataFrame(data)
+
+        new_dataframe = pandas.DataFrame(
```

Comment:
call it profileOfVariableDataframe

Reply:
Also, change version to 5.2.0 in setup.py and check it and then make the library